SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/StringTests.swift

```diff
@@ -46,5 +46,26 @@ class StringTests: XCTestCase {
             "hi"
         )
     }
+
+    func testRustStrEqualityOperator() throws {
+        XCTContext.runActivity(named: "Should be equal"){
+            _ in
+            let hello1 = create_string("hello")
+            let hello2 = create_string("hello")
+            XCTAssertEqual(hello1.as_str(), hello2.as_str())
+        }
+        XCTContext.runActivity(named: "Should not be equal"){
+            _ in
+            //Equal length
```

Comment:
minor: The `Equal length` and `Not equal length` are on the wrong tests. Need to switch them.